crates/astria-sequencer-relayer/src/metrics_init.rs

```diff
@@ -96,3 +108,10 @@ pub const SEQUENCER_HEIGHT_FETCH_FAILURE_COUNT: &str = concat!(
     env!("CARGO_CRATE_NAME"),
     "_sequencer_height_fetch_failure_count",
 );
+
+pub const TOTAL_ASTRIA_BLOB_DATA_SIZE_FOR_BLOCK: &str =
+    concat!(env!("CARGO_CRATE_NAME"), "_total_astria_blob_data_size");
```

Comment:
There's a little inconsistency around the name/description here.  In the `describe_gauge!` we have `_DATA_SIZE_FOR_BLOCK` and "for a Astria **blob**", but here we have just `_data_size`.